config/static_test.go

```diff
@@ -26,6 +26,8 @@ Bro:
     DBRoot: "RITA"
     MetaDB: MetaDatabase
     ImportBuffer: 100000
+UserConfig:
+    UpdateCheckFrequency: 14
```

Comment:
This should have been added before.  But since I changed code related to this feature this is a good sanity check that nothing is broken.